sonar-plugin-api/src/main/java/org/sonar/api/server/rule/RulesDefinition.java

```diff
@@ -925,6 +927,17 @@ private void validate() {
       }
     }
 
+    /**
+     * Register a repository and key under which this rule used be known.
```

Comment:
@sns-seb I think you should cross-reference these 2 new methods' in javadocs

Reply:
added